src/test/typescript-worker_test.ts

```diff
@@ -591,6 +591,72 @@ suite('typescript builder', () => {
     await checkTransform(files, expected, {}, cdn);
   });
 
+  /* JSX Compilation */
```

Comment:
I think we don't need this comment, the test already has a human readable description.

Reply:
:) sounds good, done!